conf/st2.dev.conf

```diff
@@ -106,3 +106,6 @@ pack_group = stanley
 [mistral]
 v2_base_url = http://127.0.0.1:8989/v2
 jitter_interval = 0
+
+[packs]
+enable_common_libs = True
```

Comment:
We have `[system]` where we have other packs related options there so there are some inconsistencies.

Anyway, I'm fine with this for now.